src/components/metadata/index.js

```diff
@@ -42,9 +42,14 @@ const MetaData = ({
   const hasCode = Boolean(metadata?.code);
   const showCodePanel = visible && visibleCode && hasCode;
   const showCodeSwitch = hasCode;
+  const runCommand = Boolean(metadata?.runCommand)
+    ? metadata.runCommand
+    : isTaskNode
+    ? 'Please provide a name argument for this node in order to see a run command.'
```

Comment:
Looks like a ternary operator inside a ternary? In this case it's looking tricky to read, I think split into two variables with one ternary each might help here.